Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/PointDrawing.cs

```diff
@@ -111,11 +119,14 @@ static void UpdatePoints(List<Vector3> points, Vector3? point)
         {
             if (point.HasValue)
             {
+                var localPosition = s_InvTransform.MultiplyPoint(point.Value);
+                if (s_CurrentPointIndex > 0 &&
+                    (localPosition - points[s_CurrentPointIndex - 1]).sqrMagnitude < kDistanceEpsilon)
+                    return;
```

Comment:
Ignore any point that's the same as the last point when doing free-draw/rectangle drawing